pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -43,13 +43,15 @@ include::../modules/con_definition-of-metadata.adoc[leveloffset=+2]
 
 include::../modules/ref_identifying-module-types.adoc[leveloffset=+2]
```

Comment:
This file seems to be missing or it has been renamed. It could be ref_adding-module-types_to_modules_in_Pantheon.adoc

Reply:
Thanks. It's already changed.